.github/workflows/docusaurus.yml

```diff
@@ -5,14 +5,17 @@ on:
       - master # default branch name for each repo that docs should be push from
```

Comment:
@arnautov-anton I think you have to add `develop` here as well to trigger the action

Reply:
Yes! Missed that one, thank you, Zita! :)